common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1,4 +1,10 @@
 {
+  "core.wynntils.category.inventory": "Inventory",
```

Comment:
woudln't feature.wynntils.category be better? As these are feature categories

Reply:
Fine either way by me, though.

Reply:
That was my first thought as well, but we have `feature.wynntils.<NAME>` as a pattern, where this is a reserved namespace for feature `<NAME>`. While we might never have a feature named CategoryFeature, I still felt it was mixing two different things.